core/server/master/src/main/java/alluxio/master/block/meta/StaticWorkerMeta.java

```diff
@@ -29,7 +29,10 @@
   final long mId;
   /** Start time of the worker in ms. */
   final long mStartTimeMs;
-
+  /** the start time of version. */
+  long mStartTime;
+  /** the version of worker. */
+  String mVersion;
```

Comment:
This class is intended to be static and therefore thread-safe. So please make this `final`.

Reply:
Again this class is immutable and fields should be final

Reply:
Because of the limitation of the fianl keyword, these data cannot be stored. We can store the data in a newly created class or in MasterWorkerInfo. What do you think?

Reply:
@codings-dan Yes having a separated class will be a good way. Maybe `WorkerTimeMeta` can be a candidate name. In order to avoid yet another lock on this object (which makes it even more complicated to acquire the necessary locks), you can either guarantee thread safety within this class, or use an `AtomicReference<WorkerTimeMeta>` and create one more object each time any time field is updated.